index.html

```diff
@@ -175,20 +290,19 @@ <h2 class="about poppins">About Myself</h2>
                         <p class="css3 poppins">CSS</p>
                     </div>
                 </li>
-            </div>
-        </ul>
-        </div>
-        <hr/>
-        <div class="frameworks">
+            </ul>
+            <hr/>
+            <div class="frameworks">
                 <p class="frame poppins">Frameworks</p>
                 <button name="button-arrow2" type="button"><img class="arrow-right" alt="" width="512px" height= "512px" src="images/right-arrow.png"></button>
-        </div>
-        <hr/>
-        <div class="skills">
+            </div>
+            <hr/>
+            <div class="skills">
                 <p class="skill poppins">Skills</p>
                 <button name="button-arrow3" type="button"> <img class="arrow-right" alt="" width="512px" height= "512px" src="images/right-arrow.png"></button>
+            </div>
+            <hr/>
         </div>
-        <hr/>
     </section>
     <footer id="foot">
         <section id="contact-form">
```

Comment:
- [ ] Please consider adding the background color for the `contact section, currently, your design has a white background which does not transparent with the text color. Remember as a frontend engineers we are expected to deliver almost identical webpages.

### Your design as of now,
![image](https://user-images.githubusercontent.com/69077061/230749809-5f1ce951-482b-4863-b460-094dd5d87b8d.png)

Reply:
I'm sorry, but my design in my computer is showing both, the background for the heading and the background for the contact form, for both, the mobile and the desktop version. Do you mean another background?